Revise bullet to clarify offline install option, bare-metal-install-desktop (#925)
clearlinux/clear-linux-documentation: source/get-started/bare-metal-install-desktop.rst

   * Check marks indicate a selection has been made.

   * The installer image contains the default bundles required for  

>      installation. Off-line installation is possible if you do not install

Reviewer: ```suggestion
     installation. An Internet connection is only required if you install
```

>      additional bundles. 

Reviewer: ```suggestion
     additional bundles from the Advanced options. 
```
